static/src/javascripts/projects/common/modules/video/metadata.js

```diff
@@ -1,4 +1,5 @@
 import config from 'lib/config';
+import fetch from 'lib/fetch';
```

Comment:
Just out of interest, what was the issue here?

Reply:
There needs to be a mock for node-fetch. It’ll come through the identity API conversion to Typescript.